.github/workflows/deploy.yml

```diff
@@ -10,10 +10,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: Set up Node.js 14.x
+      - name: Set up Node.js 13.x
         uses: actions/setup-node@v1
         with:
-          node-version: 14.x
+          node-version: 13.x
```

Comment:
Maybe we should use version 12 instead of 13, because 13 is not supported anymore: https://nodejs.org/en/about/releases/

12 will be supported until april 2022.

In the meatime, I think we should do what r4mmer suggested, so we are able to move to newer Node.js versions in the future.

Reply:
Makes sense. Changed here: https://github.com/HathorNetwork/hathor-explorer/pull/144/commits/2959467fe44dc9f1fb54c5ae887a7823b266d2ad and created an issue because of @r4mmer suggestion (https://github.com/HathorNetwork/hathor-explorer/issues/145)